From eb35eb763c15e8e7f8123fa2616d31339e1180d7 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Wed, 14 Feb 2024 18:56:10 +0000
Subject: IWYU: add missing include for std::optional usage in big_endian.h

Bug: 41455655
Change-Id: I52806fa7e8a0c2d7b263a9090b64c34b53b11092
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5296116
Commit-Queue: Kyle Charbonneau <kylechar@chromium.org>
Reviewed-by: Kyle Charbonneau <kylechar@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1260608}
---
 base/big_endian.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/base/big_endian.h b/base/big_endian.h
index 98edea511ae9d..8be510f6e90b3 100644
--- a/base/big_endian.h
+++ b/base/big_endian.h
@@ -8,6 +8,7 @@
 #include <stddef.h>
 #include <stdint.h>
 #include <string.h>
+#include <optional>
 #include <type_traits>
 
 #include "base/base_export.h"
-- 
2.40.1

